pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -102,7 +102,10 @@ angular: teacherDashboard
       //  'remove': {method:'DELETE'},
       //  'delete': {method:'DELETE'}
          update: {method:'POST', url: 'v2/sections/:id/update'},
+         // TODO: This isn't very restful
+         allStudents: {method:'GET', url:'v2/sections/:id/students', isArray: true},
          addStudents: {method:'POST', url:'/v2/sections/:id/students', isArray: true},
+         moveStudents: {method:'POST', url:'/sections/:new_section_code/transfers'},
```

Comment:
In production we proxy apis like this so they can be accessed from code.org and studio.code.org to avoid cross domain security checks -- if you can't get that to work for that reason I can set that up for you